Portfolio: Setup

peterdtitan opened the pull request:
Started working on Portfolio Website Project. Made additions and changes to project structure. The following files were added and/or removed: 

- index.html was created and necessary header and body content were added.
- style.css was created and linked to index.html.
- page media assets folder was created.
- correct configuration of the project structure (adding linters) was set up.

Commit:
Setup Project Structure

Commit:
Update README.md

Commit:
Add html and css files

Commit:
Update html and css files

Commit:
Update css styling and html content

Commit:
Update README.md

Commit:
Fix styling errors

youmari reviewed (changes requested):
Hi @peterdtitan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] doesn't have a descriptive title and short summary that tells in more detail what you've done in PR
- [ ] doesn't have linters set up correctly please try to fix that  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Commit:
Remove .gitignore file

Commit:
Remove .gitignore file

Commit:
Fix linter errors

Commit:
Fix linter errors

Commit:
Fix linter errors

Kakalanp commented on index.html:
```diff
@@ -0,0 +1,40 @@
+    <header>
+        <div class="topnav">
+            <a href="#home" class="logo">Peter Okorafor</a>
+        </div>
```
Remember to add a `<nav>` element for your hamburger button, that way, navigation will be much easier for the user :wink: 

![image](https://user-images.githubusercontent.com/78335447/170393222-69bddef1-7450-4c02-b7e4-c852eb691779.png)

Kakalanp commented on index.html:
```diff
@@ -0,0 +1,40 @@
+        </div>
+    </header>
+
+    <div class="container">        
```
Remember it is considered good practice to use semantic tags, to follow best practices you should separate sections with `<section>` tags :fire:

peterdtitan replied:
Okay. Change made!  👍🏽

Kakalanp commented on index.html:
```diff
@@ -0,0 +1,40 @@
+    </header>
+
+    <div class="container">        
+        <p class="container headline">Hey There.<br>I'm Peter</p>
```
Kindly change your title tag to `<h1>`, the main title of a page should always use `<h1>` to follow good practices :100:

peterdtitan replied:
Should have seen that too 😄

Kakalanp commented on index.html:
```diff
@@ -0,0 +1,40 @@
+
+    <div class="container">        
+        <p class="container headline">Hey There.<br>I'm Peter</p>
+        <p class="container job-title">I am a Software Developer</p>
```
Kindly change your subtitle tags to `<h2>`, the subtitles of a page should always use `<h2>`,`<h3>`,`<h4>`,`<h5>` in a hierarchical way to follow good practices :sunglasses:

peterdtitan replied:
Great tip! Thanks.

Kakalanp commented on index.html:
```diff
@@ -0,0 +1,40 @@
+
+
+
+    <div class="links">
```
Don't forget that it's also considered good practice to use `<ul>` tags for lists, the links and `<a>` tags are fine, but you could try to contain them in different `<li>`'s to make sure your project looks professional :briefcase:

Kakalanp commented on css/style.css:
```diff
@@ -0,0 +1,78 @@
+  text-decoration: none;
+}
+
+.topnav {
```
[OPTIONAL]

You could also select the HTML elements from the `<header>` tag, that way you wouldn't need to use a new div and class :thinking:

Kakalanp reviewed (changes requested):
# Hello there @peterdtitan! <img src="https://user-images.githubusercontent.com/1303154/88677602-1635ba80-d120-11ea-84d8-d263ba5fc3c0.gif" width="28px" alt="hi">

### Good job so far! :tada: 

There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand: 

<img src="https://media.giphy.com/media/Qaxd5XpgDWMso/giphy.gif" alt="so close">

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Good implementation from the Figma design :monocle_face: 

## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Update html file

Commit:
Make changes to styling

Commit:
Added update page screenshot to README.md

Kossi-stack commented on index.html:
```diff
@@ -0,0 +1,66 @@
+        </header>
+    </section>
+
+    <section class="main-content">
```
- [x] kindly align your content well according to figma design.
- Figma Design
![Captureinli](https://user-images.githubusercontent.com/81444338/170432306-b87d7d00-c404-4f38-a19a-c5b9d2fd4769.PNG)
- Your Design
![Captureinl](https://user-images.githubusercontent.com/81444338/170432399-a6904389-9a53-49af-be9a-36669d397231.PNG)

peterdtitan replied:
For the links to external portfolios, isn't it appropriate to only add links I have a portfolio on (github, linkedin and twitter)?

Kossi-stack commented on README.md:
```diff
@@ -1 +1,51 @@
+> This is the mobile version of a personal portfolio page that shows off my accomplishments as a software developer.
+
+![screenshot](./assets/screenshot.png)
+
```
- [ ] [OPTIONAL] Kindly remove empty lines for a clean code

Kossi-stack reviewed (changes requested):
## Hi @peterdtitan ,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966](https://user-images.githubusercontent.com/81444338/170121142-a9f93bb7-38bb-45cc-8327-aab2501ef90a.gif)

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. 
 - [ ] Please make sure to add .gitignore file to the praject also don't forget to ignore node_modules.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Commit:
Changed content alignment, updated html and README.md

peterdtitan commented:
Updated content alignment to match Figma designs. The following changes were made: 

- Updated the css styling to align contents correctly as in given design.
- Added Medium and AngelList icons/links.
- Updated README file and cleaned up empty lines.

sja-thedude reviewed (approved):
## Project Approved 👍 

Hi @peterdtitan, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![giphy](https://user-images.githubusercontent.com/70269507/159485323-11dfc0c5-bd53-4956-b55a-2fc42f7a3845.gif)

Cheers and Happy coding!👏👏👏

## Optional: It is highly suggested to change your PR title from `Feature` to `Portfolio: Setup`. I tried Contacting you on slack but you have not responded. As this was the only thing that's the reason I approved but make sure to change your PR title.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
